### ISSUE TITLE
[Bug] Invalid file type when importing back up from mobile tachiyomi to tachi-desk

### ISSUE THREAD

**Issue by Eyeless-Jacks** [AriaMoradi/Tachidesk#791]
## Device information
Tachidesk version: v0.7.0
Server Operating System: Windows 11
Server Desktop Environment: N/A
Server JVM version: bundled with win64
Client Operating System: Windows 11
Client Web Browser: Google Chrome

## Steps to reproduce
1. Make backup from tachiyomi on android
2. get the backup file from phone to PC
3. drag the file to restore backup in tachi-desk

### Expected behavior
i saw a video showing that this could send my manga and history from Tachiyomi to tachi-desk once I just drag my backup file to the restore backup

### Actual behavior
Instead it said Invalid file type but some comments on the video said it was working, is there an easy way to get my phone manga to tachidesk

## Other details
this is the video I saw https://www.youtube.com/watch?v=GOrGHXSryts

**Comment by Robonau** [comment]
if you are on release youl have to change the file extension to `.proto.gz`
if you are on preview the the `.tachibk` file u get from tachiyomi should work

**Comment by Eyeless-Jacks** [comment]
I've tried it and it still say invalid file type heres the file I trying to send
[tachiyomi_2023-05-10_16-04.proto.gz.gz](https://github.com/Suwayomi/Tachidesk-Server/files/13642098/tachiyomi_2023-05-10_16-04.proto.gz.gz)

**Comment by Robonau** [comment]
how did it become `gz.gz`?
if renaming it to `.proto.gz` doesn't work then u may need to extract it one layer (using winrar, 7zip or whatever method u have)

**Comment by SwissGipfel** [comment]
> if you are on release youl have to change the file extension to `.proto.gz` if you are on preview the the `.tachibk` file u get from tachiyomi should work

Tried it with `Tachidesk v0.7.0 revision r1439` in your provided docker image, but I got the same error when restoring from a `.tachibk` file. What I find odd is, that the application.log doesn't seem to notice the restore attempt even though your preview container starts in debug mode. It's just the webUI that gives me a red pop-up `invalid file type!`

Once I rename the file and give it a `.proto.gz` file ending, the restore works smoothly. I can see the log output in application.log too.
